supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -2,6 +2,14 @@
 [[general-formatting]]
 = General formatting
 
+[[date-formats]]
+== Date formats
+
+Red Hat writers follow _The IBM Style Guide_ for date formats. This list enumerates exceptions:
+
+* When the format, `day Month year`, causes a presentation or clarity issue, use instead: `Month day, year` (for example, October 3, 2019). (Issue link:https://github.com/redhat-documentation/supplementary-style-guide/issues/14[#14])
```

Comment:
* The two commas surrounding `day Month year` are not needed.
* No need to reference the GitHub issue, so you can remove the "(Issue link:https://github.com/redhat-documentation/supplementary-style-guide/issues/14[#14])" part.

Reply:
Oops! Missed this. See third commit.